dvc/repo/experiments/save.py

```diff
@@ -11,6 +13,29 @@
 logger = logging.getLogger(__name__)
 
 
+def _save_experiment(
+    repo: "Repo",
+    baseline_rev: str,
+    force: bool,
+    name: Optional[str],
+    include_untracked: Optional[List[str]],
+) -> str:
+    stages = repo.commit([], force=force)
+    exp_hash = BaseExecutor.hash_exp(stages)
```

Comment:
Names are now always generated beforehand in `exp run` and don't rely on `hash_exp`, we should probably do the same thing here

https://github.com/iterative/dvc/commit/bb7512687ce3a71bdf84f79d87e740159bb51256